README.md

```diff
@@ -145,21 +143,21 @@ welcome:
 ```php
 <?php
 
-use BeSimple\I18nRoutingBundle\Routing\I18nRouteCollectionBuilder;
+use BeSimple\I18nRoutingBundle\Routing\I18nRouteCollection;
 use Symfony\Component\Routing\Route;
-use Symfony\Component\Routing\RouteCollection;
 
-$builder = new I18nRouteCollectionBuilder();
+$builder = new I18nRouteCollection();
```

Comment:
@HeahDude Thanks for reviewing as you can see in commit 95dbc653ada3c118fd6e5ef5b1ec88b94664f186 the routebuilder is gone and we have a nice generator for this now.